ui/pages/send/send.js

```diff
@@ -111,7 +112,6 @@ export default function SendTransactionScreen() {
               `sendFlow - User pasted ${text} into address field`,
             ),
           );
```

Comment:
This update recipient call is being removed because pastes also trigger the change listener on the input in the EnsInput component, which results in `onValidAddressTyped` being called, which (as you can see above) also calls `updateRecipient`. We don't want to call that twice.